drift/lib/src/ffi/database.dart

```diff
@@ -1,7 +1,6 @@
 import 'dart:io';
 
 import 'package:meta/meta.dart';
-import 'package:sqlite3/common.dart';
```

Comment:
Are you on an older Dart or Flutter version by any chance? In the latest `sqlite3` version, the `dispose` method was moved from `Database` to `CommonDatabase`  and I was getting lints about an unresolved comment reference. 

I've forgotten to update the constraint in the pubspec though, I'll fix that.

Reply:
Ah, I just saw your comment. But with `sqlite3` version 1.6.1 importing `common.dart` should work though.

Reply:
I don't know what's the problem actually. Test simply not compiled. Can you fix this and resolve conflicts?

Reply:
The CI works now so I think it should be fine? For me, the tests in `drift` and `drift_dev` work locally too (I'm on Dart 2.16.2). Let me know if you still have problems with them.